db/migrate/20181201000100_create_message_recipients.foodsoft_messages.rb

```diff
@@ -0,0 +1,49 @@
+class CreateMessageRecipients < ActiveRecord::Migration
+  class Message < ActiveRecord::Base
+    has_many :message_recipients
+    has_many :recipients, through: :message_recipients, source: :user
+  end
+
+  def up
+    create_table :message_recipients do |t|
+      t.references :message, index: true, null: false
+      t.references :user, null: false
```

Comment:
`add_reference` [mentions](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference):
> `:index` defaults to true

Maybe add `index: false`?

Reply:
on my machine it seams that the default is `false`. at least the index is missing if there is not `index: true`

Reply:
:+1: that must have changed then after Rails 4